transport_and_format.asciidoc

```diff
@@ -33,7 +33,9 @@ The literal 'Token' indicates that the token-based authentication mechanism is u
 These are different 'tokens' than the <<mod_tokens.asciidoc#mod_tokens_token_object,Tokens>> exchanged via the <<mod_tokens.asciidoc#mod_tokens_tokens_module,Token Module>>: Tokens used by drivers to authorize charging.
 To prevent confusion, when talking about the token used here in the HTTP Authorization header, call them: 'Credentials Tokens'.
 
-Its parameter is a string consisting of printable, non-whitespace ASCII characters. It contains the 'credentials token' Base64 encoded.
+After the literal 'Token', there SHALL be a space (U+0020), followed by the credentials token encoded with Base64 according to <<https://datatracker.ietf.org/doc/html/rfc4648#section-4,RFC 4648>>.
```

Comment:
We could improve on this by stating that the token must first be encoded using a US-ASCII compatible encoding (such as UTF-8 and ISO-8859-1) and the resulting octet sequence must then be base64 encoded. This is very similar to how Basic authentication defines the encoding. In pseudo-code this will look like this:
```
let header_token_value = base64(encode("my-super-secret-ocpi-token", UTF8))
```